mypy/ipc.py

```diff
@@ -48,25 +48,24 @@ class IPCBase:
     connection = None  # type: _IPCHandle
 
     def __init__(self, name: str, timeout: Optional[float]) -> None:
-        self.READ_SIZE = 100000
         self.name = name
         self.timeout = timeout
 
-    def read(self) -> bytes:
+    def read(self, size: int = 100000) -> bytes:
```

Comment:
I'm not sure if this is worth putting in the interface, since it doesn't modify any visible behavior.

Reply:
Alright, sure